stable/insights-agent/templates/rbac-reporter/_job.yaml

```diff
@@ -0,0 +1,41 @@
+{{- define "rbacreporter.job.spec" -}}
+restartPolicy: Never
+serviceAccountName: {{ include "insights-agent.fullname" . }}
+volumes:
+- name: output
+  emptyDir: {}
+containers:
+- name: rbac-reporter
+  image: "{{ .Values.rbacreporter.image.repository }}:{{ .Values.rbacreporter.image.tag }}"
+  imagePullPolicy: Always
+  args:
+    - cluster
+    - -u
+    - https://charts.fairwinds.com/stable
+    - -u
+    - https://charts.fairwinds.com/incubator
+    - -u
+    - https://kubernetes-charts.storage.googleapis.com
+    - -u
+    - https://kubernetes-charts-incubator.storage.googleapis.com
+    - -o
+    - file
+    - --file-path
+    - /output/rbac-reporter.json
+  volumeMounts:
+  - name: output
+    mountPath: /output
+  resources:
+    {{- toYaml .Values.rbacreporter.resources | nindent 4 }}
+  securityContext:
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    runAsNonRoot: true
+    runAsUser: 1200
```

Comment:
Does it have to run as a specific user? If not it already can't run as root, so specifying a particular user seems unnecessary.